py_neuromodulation/nm_stream.py

```diff
@@ -134,23 +140,38 @@ def _run(
                 data_batch.astype(np.float64)
```

Comment:
I realized we don't need this, mne_lsl returns the data in `np.float64` format and `astype` actually makes a whole copy of the data, which is not good for performance